config/crd/bases/atlas.mongodb.com_atlasdeployments.yaml

```diff
@@ -368,6 +368,34 @@ spec:
                     type: array
                   rootCertType:
                     type: string
+                  searchNodes:
+                    description: List of settings that configure the search nodes
+                      for your cluster
+                    items:
+                      properties:
+                        instanceSize:
+                          enum:
+                          - S20_HIGHCPU_NVME
+                          - S30_HIGHCPU_NVME
+                          - S40_HIGHCPU_NVME
+                          - S50_HIGHCPU_NVME
+                          - S60_HIGHCPU_NVME
+                          - S70_HIGHCPU_NVME
+                          - S80_HIGHCPU_NVME
+                          - S30_LOWCPU_NVME
+                          - S40_LOWCPU_NVME
+                          - S50_LOWCPU_NVME
+                          - S60_LOWCPU_NVME
+                          - S80_LOWCPU_NVME
+                          - S90_LOWCPU_NVME
+                          - S100_LOWCPU_NVME
+                          - S110_LOWCPU_NVME
+                          type: string
+                        nodeCount:
+                          maximum: 32
+                          type: integer
```

Comment:
These does not seem to fit what we had in the docs:

**`instanceSize`** there was an expression of `{item} | |{item}` where each item was an enum with the values listed here. But here we are just saying instance size is a single `{item}`of that enum.

Also the **`nodeCount`** in examples was something like `1..32` which is a string representing a range. To get the same we would need to either user a string and parse the ranges such as `1`, `3..12`, or `..23`. Or we should provide a **min** and **max** integers with default values maybe.

Reply:
ah ok, I got it now. Thanks for explaining!